### PR TITLE
Upgrade Liquibase from 3.5.5 to 3.6.0, to be compatible with Jdk10.

### PR DESCRIPTION
**Pull request by xiaodong-xie**
###### Problem:
Liquibase 3.5.5 cannot run on Jdk10, due to its JDK version check failure. The issue was fixed in 3.6.0. 

###### Solution:
Bump up version of Liquibase from 3.5.5 to newest 3.6.0. 

###### Result:
Dropwizard could run on Jdk10.

### CODE REVIEW

**Comment by nickbabcock** [comment]
Oh how exciting, this would fix #1708 too

**Comment by xiaodong-xie** [comment]
I'll look into those test failures...

**Comment by xiaodong-xie** [comment]
https://github.com/liquibase/liquibase/commit/9acf2fb33208ebd0673ef7b5f3f63ddc585194bb#diff-df4fa5e5aa8d525d2a3caaaa237ec2df

Liquibase updated `CHECKSUM_ALGORITHM` from 7 to 8.

**Comment by xiaodong-xie** [comment]
Changed contents in `test-db.mv.db` file, due to the `CHECKSUM_ALGORITHM` change in Liquibase.

```
7:0f9e4aa31e065bbdffa9eb3ead0ae2c8 -> 8:b0ec2153e830053427f3182dfa7ae6dd
7:3a61a7a72c9ce082b7059215975e6e09 -> 8:0f3683b37321ccfb1694a044986de4d9
7:aad8d260150f46b5a4663fc4b2a80474 -> 8:d3ff16527dcfc0c86495d15619027d5e

```

File: dropwizard-migrations/src/test/java/io/dropwizard/migrations/AbstractMigrationTest.java [hunk 1]
```diff
@@ -10,6 +12,7 @@
 
     static {
         ArgumentParsers.setTerminalWidthDetection(false);
+        SqlGeneratorFactory.getInstance().unregister(AddColumnGeneratorSQLite.class);
```
Comment (xiaodong-xie) [inline dropwizard-migrations/src/test/java/io/dropwizard/migrations/AbstractMigrationTest.java]: Created a PR against Liquibase https://github.com/liquibase/liquibase/pull/756. Before the bug is fixed, we have to add this workaround.

**Comment by xiaodong-xie** [comment]
Ping @jplock, finally I made the build pass. 😄

**Comment by jplock** [comment]
Thanks!

**Comment by xiaodong-xie** [comment]
Thanks for merging, @jplock. 👍 😄

**Pull request merged**